src/extension/manifest.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "imgix Salesforce B2C Commerce Cloud Extension",
+  "version": "0.0.1",
+  "manifest_version": 2,
+  "description": "This extension provides a Salesforce Commerce Cloud B2C integration with imgix.",
+  "homepage_url": "http://imgix.com",
```

nit: should this be `https://`?
```suggestion
  "homepage_url": "https://imgix.com",
```